llvm/test/Transforms/PGOProfile/memprof_annotate_indirect_call.test

```diff
@@ -0,0 +1,89 @@
+; RUN: split-file %s %t
+
+; COM: Basic functionality with flag toggle
```

Comment:
Why use COM here and below? Just the leading ";" means it should be interpreted as a comment, although I typically use ";;" to set off the comment. Typically COM is only needed to prevent FileCheck from misinterpreting a reference to a directive.